hermes/probe/mock/mock_gcs.go

```diff
@@ -0,0 +1,88 @@
+package mock
```

Comment:
Add license header and package description comment.

Reply:
Added as a TODO (#70 )